velox/docs/functions/spark/string.rst

```diff
@@ -427,3 +427,15 @@ String Functions
     Returns string with all characters changed to uppercase. ::
 
         SELECT upper('SparkSql'); -- SPARKSQL
+
+.. spark:function:: varchar_type_write_side_check(string, limit) -> varchar
+
+    Removes trailing 0x20(space) characters from ``string`` to fit in the ``limit``, if not fit, throws exception.
```

Comment:
Could you clarify if some spaces are removed till the limit is met, or all the spaces are removed?